homebridge-ui/public/index.html

```diff
@@ -173,17 +178,42 @@ <h5 class="mb-0 text-center">
       } catch (e) {
         homebridge.toast.error(e.message);
         homebridge.hideSpinner();
+        // Abort if we did not login successfully.
+        return;
       }
 
       const table = document.getElementById('discoverTable').getElementsByTagName('tbody')[0];
       table.innerHTML = '';
 
+      // Merge current config over to the server so discover has latest device config...
+      let currentConfig = await homebridge.getPluginConfig();
+      currentConfig = currentConfig.length > 0 ? currentConfig[0] : {};
+      configuration = await homebridge.request('/mergeToDefault', { config: currentConfig });
+
+      // Add any existing IP addresses to list user provided
+      Object.values(configuration.devices).forEach((d) => {
```

Comment:
I've seen this pattern when you wrap arrays with Object.values. Is there any reason why you doing that?

Reply:
Oversight on my part, keep forgetting that devices is already an array.  I have fixed.